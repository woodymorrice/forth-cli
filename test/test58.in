."helloworld" ."hello" space ."world" + + constant
."helloworld" ."hello" space ."world" + + constant . . .
