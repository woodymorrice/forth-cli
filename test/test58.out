hello world
helloworld
stack empty
